apps/web/lib/features/integrations/boards/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const FILE_CACHE_MAX_AGE_SEC = 31536000;
+
+export const FILE_UPLOAD_MAX_BYTES = 3 * 1024 * 1024;
+
+export const LOCAL_STORAGE_THEME = 'excalidraw-theme';
```

Comment:
@paradoxe35 can we have `board-theme` here or it is managed by excalidraw?

Reply:
If we need for that to rename too many things in the original Excalidraw code / do many changes, let's not do it. I would prefer to make as little changes to original code as possible so we can easy pull the latest changes any moment and merge to our repos!